Makefile

```diff
@@ -42,36 +54,34 @@ check-noextensions:: clean
 check-contrib:: clean
 	$(RUNTEST) -v dulwich.contrib.test_suite
 
-check-all: check check-pypy check-noextensions
-
 typing:
 	$(PYTHON) -m mypy dulwich
 
-clean::
-	$(SETUP) clean --all
-	rm -f dulwich/*.so
-
 style:
 	$(RUFF) check .
 
+fix:
+	ruff check --fix .
+
+format:
+	ruff format --check .
```

Comment:
This is a change in behaviour, please just have "make format" do formatting - not just check whether reformatting is necessary.

Reply:
But that's what `reformat` already does? This is a new target that only does checks, but doesn't change anything, to provide parity with other check/fix targets.

Reply:
Oh, I see. Having both format and reformat where one makes changes confused me. Don't really want to bikeshed on this and happy to merge as-is, but I don't think it's a great name.